### ISSUE TITLE
Key selection dropdown in instrument list not accessible by keyboard

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#19052]
### Issue type

UI bug

### Bug description

In the instrument list, when an instrument is selected that has multiple transposition options such as clariney, the dropdown list to select the key cannot be operated by keyboard.

### Steps to reproduce

1. Create new score
2. Type Clarinet into search
3. Try to select the "A" clarinet using keyboard alone

Result: there is no way to focus on the dropdown.  I assume that if this is fixed, the screen reader will also reader the selected key, but of course that should be checked as well.

### MuseScore Version

OS: Debian GNU/Linux 11 (bullseye), Arch.: x86_64, MuseScore version (64-bit): 4.1.1-232071203, revision: github-musescore-musescore-e4d1ddf

### Regression

Yes, this used to work in MuseScore 3.x and now is broken

### Operating system

Debian 11

### Additional context

MuseScore didn't use a dropdown; it simply listed the instruments individually.  So bottom line, it was possible to add clarinets in keys other than Bb with MuseScore 3, but is not in MuseScore 4.

This is exacerbated by the fact that staff/part properties is also inaccessible - see https://github.com/musescore/MuseScore/issues/15379 and https://github.com/musescore/MuseScore/issues/18426.  So what would otherwise be the workaround of changing transposition after adding a Bb clarient is not an option for users who are blind, either.

**Comment by bkunda** [comment]
@RomanPudashkin could this potentially be one for @mathesoncalum?

**Comment by DmitryArefiev** [comment]
Implemented in #20591

**Comment by DmitryArefiev** [comment]
After https://github.com/musescore/MuseScore/pull/20591 implementation there is one navigation issue left. I've logged it separately as #20657
